src/input/ButtonConfigWidget.cc

```diff
@@ -61,6 +62,12 @@ void ButtonConfigWidget::init() {
 			initComboBox(box,*ent);
 		}
 	}
+
+	WheelIgnorer *wheelIgnorer = new WheelIgnorer();
```

Comment:
This object should probably receive a parent so it gets deleted at some point.
You can also `auto` this for less repetition: `auto wheelIgnorer = new WheelIgnorer();`

Reply:
Thank you, that makes sense. I now set the ButtonConfigWidget and AxisConfigWidget as parents respectively.

Reply:
On this note, you should be able to pass this to the `WheelIgnorer` contructor; that's the common way of assigning parents in Qt. Adding a constructor also makes it clear that this is how the object is supposed to be used (e.g. you could require an argument to be passed).